components/eamxx/src/physics/register_physics.hpp

```diff
@@ -51,7 +51,7 @@ inline void register_physics () {
   proc_factory.register_product("Nudging",&create_atmosphere_process<Nudging>);
 #endif
 #ifdef EAMXX_HAS_MAM
-  proc_factory.register_product("MAMMicrophysics",&create_atmosphere_process<MAMMicrophysics>);
+  proc_factory.register_product("MAM4Microphysics",&create_atmosphere_process<MAMMicrophysics>);
```

Comment:
Should this string be `mam4_micro`, to match the output of the `name()` function?

Reply:
If I had my way, this process would have a single name everywhere. I don't really understand the reasoning for all the names I see floating around in the code. I believe we can fix this in the next PR.